transformation/src/handlers/storageHandler.ts

```diff
@@ -0,0 +1,307 @@
+import { Connection, ConnectionOptions, createConnection, getConnection, Repository, DeleteResult, UpdateResult, FindConditions } from 'typeorm';
+import { TransformationConfig } from '../models/TransformationConfig';
+import { TransformationRepository } from '../interfaces/transformationRepository';
+import { PipelineMetaData } from '../models/PipelineMetaData';
+import TransformationData from '../models/TransformationData';
+
+
+
+/**
+ * This class handles Requests to the Nofification Database 
+ * in order to store and get Notification Configurations.
+ */
+export class StorageHandler implements TransformationRepository{
+
+    private configRepository!: Repository<TransformationConfig>
+    private dbConnection!: Connection | null
```

Comment:
Never used?

Reply:
I had to assign null due to result of a `createConnection()` - Call. The `!` is a result of not initializing it in the constructor.
Unfortunately I have no clue to achieve this in another way.